<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pesq_cat</class>
 <widget class="QWidget" name="pesq_cat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pesquisar Categoria</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bt_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="bt_del">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_pesquisa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Campo de Pesquisa</string>
   </property>
   <widget class="QLineEdit" name="txt_pesquisa">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>29</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Digite o nome da Categoria</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pesquisar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_limpar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tb_categoria">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>501</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>teste</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>teste</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>asd</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>sdasd</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Pesquisar Categoria&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_pesquisa</tabstop>
  <tabstop>bt_pesquisar</tabstop>
  <tabstop>bt_limpar</tabstop>
  <tabstop>bt_edit</tabstop>
  <tabstop>bt_del</tabstop>
  <tabstop>tb_categoria</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
